From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Salyzyn <salyzyn@google.com>
Date: Wed, 28 Aug 2019 09:38:07 -0700
Subject: ANDROID: cuttlefish: overlayfs: regression

commit a77712c342a56420013625c93d8aa1501455a984
("ANDROID: Remove unused cuttlefish build infra") caused a
regression in availability of CONFIG_OVERLAY_FS configuration.

NB: The upstream work to add override_creds to overlayfs is
    required in order to pass adb-remount-test.sh.

Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Test: confirm overlay filesystem available in the build
Bug: 138649540
Change-Id: I9e51475f1025f726e69c6f513099248bf452cc2d
---
 arch/x86/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index a37fa09bd107..b3dad521ee8a 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -311,6 +311,7 @@ CONFIG_FS_ENCRYPTION=y
 CONFIG_QUOTA=y
 CONFIG_QFMT_V2=y
 CONFIG_FUSE_FS=y
+CONFIG_OVERLAY_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
